online-fishing-store/src/app/components/header/header.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeaderComponent } from './header.component';
+import { SearchComponent } from './search/search.component';
+import { ShoppingCartComponent } from './shopping-cart/shopping-cart.component';
+import { ProfileComponent } from './profile/profile.component';
+import { FormsModule } from '@angular/forms';
+
+@NgModule({
+  declarations: [
+    HeaderComponent,
+    ProfileComponent,
+    SearchComponent,
+    ShoppingCartComponent,
+  ],
+  imports: [CommonModule, FormsModule],
+  exports: [HeaderComponent],
```

Comment:
Remove this component from exports. Read more about exports field please, you need to understand more for what this field exist